lib/ios-crash-log.js

```diff
@@ -0,0 +1,68 @@
+import path from 'path';
+import _fs from 'fs';
+import _ from 'lodash';
+import glob from 'glob';
+import { getLogger } from 'appium-support';
+import B from 'bluebird';
+
+let fs = {
+  stat: B.promisify(_fs.statSync),
+  readFile: B.promisify(_fs.readFile)
+};
+require('date-utils');
+let logger = getLogger('ios-crash-log');
```

Comment:
FWIW I've been using UpperCamelCase for log names, like BaseDriver, AndroidDriver, etc... If we want to follow that, these should be IOSLog and IOSCrashLog